libs/music/docs/reference/music/play-sound.md

```diff
@@ -0,0 +1,58 @@
+# play Sound
+
+Play some notes of music you have placed in a string.
+
+```sig
+music.playSound('')
+```
+
+Your program doesn't wait for the sound to finish. Then next part of your program starts right away.
```

Comment:
We should have a single place to document the format in playSound/UntilDone. Otherwise, they will grow out of sync.